client/admin/components/users/Enrollments/EnrollmentDialog.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <v-dialog v-hotkey="{ esc: close }" v-model="show" width="700">
+    <v-form @submit.prevent="bulkEnroll">
+      <v-card class="pa-3">
+        <v-card-title class="headline">Enroll users to Program</v-card-title>
+        <v-card-text>
+          <v-autocomplete
+            v-model="programId"
+            :items="listPrograms"
+            :loading="isLoading"
+            @focus="focusTrap.pause()"
+            @blur="focusTrap.unpause()"
+            label="Program"
+            placeholder="Start typing to Search"
+            prepend-icon="search"
+            clearable
+            name="program"/>
+        </v-card-text>
+        <v-card-actions>
+          <v-spacer/>
+          <v-btn @click="close">Cancel</v-btn>
+          <v-btn color="success" type="submit">Enroll</v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-form>
+  </v-dialog>
+</template>
+
+<script>
+import enrollmentApi from '@/admin/api/enrollment';
+import forEach from 'lodash/forEach';
+import map from 'lodash/map';
+import { mapState } from 'vuex';
+import pick from 'lodash/pick';
+import { withFocusTrap } from '@/common/focustrap';
+import { withValidation } from '@/common/validation';
+
+const el = vm => vm.$children[0].$refs.dialog;
+
+export default {
+  name: 'enrollment-dialog',
+  mixins: [withValidation(), withFocusTrap({ el })],
+  props: {
+    visible: { type: Boolean, default: false },
+    userIds: { type: Array, default: () => ([]) }
+  },
+  data() {
+    return {
+      programId: null,
+      userId: null,
+      isLoading: false
+    };
+  },
+  computed: {
+    show: {
+      get() {
+        return this.visible;
+      },
+      set(value) {
+        if (!value) this.close();
+      }
+    },
+    ...mapState('programs', { programs: 'items' }),
+    listPrograms() {
```

Comment:
programList?
Maybe call this variable programs, and in the line above renamed it to something else.